packages/web-components/fast-foundation/src/combobox/combobox.spec.md

```diff
@@ -0,0 +1,149 @@
+# Combobox
+
+## Overview
+
+As defined by the [W3C](https://w3c.github.io/aria-practices/#combobox):
+> A combobox is an input widget with an associated popup that enables users to select a value for the combobox from a collection of possible values. In some implementations, the popup presents allowed values, while in other implementations, the popup presents suggested values, and users may either select one of the suggestions or type a value. The popup may be a listbox, grid, tree, or dialog. Many implementations also include a third optional element -- a graphical Open button adjacent to the combobox, which indicates availability of the popup. Activating the Open button displays the popup if suggestions are available.
+
+### Use Cases
+
+- *A customer using the component on a web page.*
+On a web page a customer is providing their mailing address for an order form. The customer clicks the state `<fast-combobox>` and sees a list of options. The customer begins typing their state and the field is automatically populated with the closest matching state. Then, the customer uses the arrow keys to highlight their state and presses `enter` to commit the value to the combobox.
```

Comment:
I find the use of the word "state" slightly confusing from a non US perspective (some countries have counties, provinces, etc.) and also from the perspective of the "state" in programming terms, so here's a suggestion just for additional clarity.

```suggestion
On a web page a customer is providing their US mailing address for an order form. The customer clicks a `<fast-combobox>` to fill in their state (California, Ohio, etc.) and sees a list of options. The customer begins typing the state name and the field is automatically populated with the closest matching state. Then, the customer uses the arrow keys to highlight their state and presses `enter` to commit the value to the combobox.
```